examples/slurm/README.md

```diff
@@ -1,50 +1,54 @@
-Moneo Job Id Update Integration w/ Slurm
-=====
-Description
+# Moneo Job Id Update Integration w/ Slurm #
+
+## Description ##
+
 -----
-Moneo allows for the assignment of job IDs to a subset of nodes. The command ```python3 moneo.py -j <jobid> -c <job host.ini file>``` accomplishes this. The command requires a host.ini file which has a specific format. It also requires a job Id.
-
-The host.ini has the following format:
-```
-host-1
-host-2
-host-3
-```
+Moneo allows for the assignment of job IDs to a subset of nodes. The running command ```/tmp/moneo-worker/jobIdUpdate.sh <jobid>``` on each compute node accomplishes this.

```

Comment:
Any reason why we keep this in /tmp/ rather than Moneo installation folder?

Reply:
Both have the same functionality. However, if a user is not using our hpc images only the /tmp location may be available if they have deployed from a remote machine.

Reply:
ok, got it. I still think it would be better to avoid /tmp. 

Can we unify it to use the HPC image path (and we put the same as default path in installation). If user sets a custom path, user has the responsibility to update this path as well. What do you think?